pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -44,23 +45,26 @@ type Synchronizer struct {
 	operationalStateChan chan<- events.OperationalStateEvent
 	key                  southbound.DeviceID
 	query                client.Query
-	operationalCache     map[string]string
 	modelReadOnlyPaths   modelregistry.ReadOnlyPathMap
+	operationalCache     map[topocache.ID]map[string]*change.TypedValue
+	modelPlugin          modelregistry.ModelPlugin
```

Comment:
I couldn't see how this is used by the synchronizer

Reply:
removed